From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Fernando Aguilar <feaguila@microsoft.com>
Date: Thu, 23 May 2019 16:19:55 -0700
Subject: [PATCH 2/7] MSBuild package versions for dotnet Source Build (#2857)

Upgraded MSBuild package deps. to 16.x; added dotnet source-build Repo API
---
 build/config.props | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/build/config.props b/build/config.props
index e45370767..59ef2b743 100644
--- a/build/config.props
+++ b/build/config.props
@@ -55,6 +55,15 @@
     <ToolsetTargetBranches Condition="'$(OverrideToolsetTargetBranches)' == ''">master</ToolsetTargetBranches>
   </PropertyGroup>
 
+  <!-- Special project import for dotnet source build
+
+  The dotnet source-build Repo API overrides *PackageVersion properties with versions that were built before.
+  Those paackages are injected in this props file
+
+  See: https://github.com/dotnet/source-build/blob/master/Documentation/auto-dependency-flow/api.md#-pdotnetpackageversionpropspathpath
+  -->
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="'$(DotNetPackageVersionPropsPath)' != ''" />
+
   <!-- Config -->
   <PropertyGroup>
     <RepositoryName>NuGet</RepositoryName>
